src/com/facebook/buck/jvm/groovy/GroovycStep.java

```diff
@@ -61,8 +69,8 @@
 
   @Override
   public int execute(ExecutionContext context) throws IOException, InterruptedException {
-
-    ProcessBuilder processBuilder = new ProcessBuilder(createCommand());
+    ImmutableList<String> command = createCommand();
+    ProcessBuilder processBuilder = new ProcessBuilder(command);
```

Comment:
Add a lint if that's the case? Not sure how external people would know and this is perfect for lint.